nextcloud_update.sh

```diff
@@ -586,6 +586,7 @@ $DOCKER_RUN_OUTPUT"
     docker_update_specific 'onlyoffice' 'OnlyOffice'
     # Full Text Search
     docker_update_specific 'fts_esror' 'Full Text Search'
+    docker_update_specific 'fts_os-node' 'Full Text Search'
```

Comment:
@Ark74 I guess this will not work for a docker-compose container, does it?

Reply:
Nope, but I add it there anyway. Will have a look later. If you want you can check as well.